app/queries/servers/team.ts

```diff
@@ -322,6 +324,15 @@ export const getTeamById = async (database: Database, teamId: string) => {
     }
 };
 
+export const getTeamSearchHistoryById = async (database: Database, id: string) => {
+    try {
+        const teamSearchHistory = (await database.get<TeamSearchHistoryModel>(TEAM_SEARCH_HISTORY).find(id));
```

Comment:
I tend to agree and created a ticket for mobile platform to investigate because they appear quite frequently.

https://mattermost.atlassian.net/browse/MM-45588